src/main/java/racingcar/controller/Winner.java

```diff
@@ -0,0 +1,21 @@
+package racingcar.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.model.Car;
+
+public class Winner {
+
+  private int getMaxLocation(List<Car> cars) {
+    return cars.stream().mapToInt(Car::getLocation).max().orElse(0);
+  }
+
+  public boolean isWinner(Car car, int maxLocation) {
+    return car.getLocation() == maxLocation;
+  }
+
+  public List<Car> getWinner(List<Car> cars) {
+    int maxLocation = getMaxLocation(cars);
+    return cars.stream().filter(car -> isWinner(car, maxLocation)).collect(Collectors.toList());
```

Comment:
작성해주신 부분을 스트림을 사용하지 않고 작성해보면 아래와 같이 표현됩니다. 
프로그래밍 요구사항에 보면
indent(인덴트, 들여쓰기) depth를 2를 넘지 않도록 구현한다. 1까지만 허용한다.라는 요구사항이있습니다.
for안에 if를 사용함으로써 2depth가 되었네요~

참고하면 좋으실 것 같아 아래 url 전달드립니다
https://developerfarm.wordpress.com/2012/01/26/object_calisthenics_2/
```suggestion
         for (Car car : cars) {
          if (isWinner(car, maxLocation)) {
              list.add(car);
          }
      }
```